docs/apis/subsystems/task/adhoc.md

```diff
@@ -156,3 +156,32 @@ The `reschedule_or_queue_adhoc_task()` function will locate any existing task ma
 ```php
 \core\task\manager::reschedule_or_queue_adhoc_task($task);
 ```
+
+#### Set the retry limit for a task
+
+Since Moodle 4.4, the maximum number of retries for a task is **12** by default.
```

Comment:
Use a `<Since>` tag here too.

https://github.com/moodle/devdocs/pull/974 makes this a global import.

```suggestion

<Since version="4.4" issueNumber="MDL-79128" />

Since Moodle 4.4, the maximum number of retries for a task is **12** by default.
```